api/search.go

```diff
@@ -124,17 +124,19 @@ func SearchHandlerFunc(elasticSearchClient ElasticSearcher) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
 		ctx := req.Context()
 		params := req.URL.Query()
-		size, err := strconv.Atoi(paramGet(params, "size", "10"))
+		sizeParam := paramGet(params, "size", "10")
+		size, err := strconv.Atoi(sizeParam)
 		if err != nil {
-			log.Debug("Expected number only characters for paramater 'size'",
-				log.Data{"Size": paramGet(params, "size", "10"), "Error": err.Error()})
+			log.Event(ctx, "search paramater 'size' provided with non numeric characters",
```

Comment:
`paramater` - spelling.

Our convention is that the event field should be a constant and not a concatenated or dynamically generated value. It should also be the same for every occurrence of the same error. While you haven't done that here you do have the 2 unique event strings for the same issue:
- "search paramater 'size' provided with non numeric characters"
- "search paramater 'from' provided with non numeric characters"

The reason we decided on this approach was:
- To make it easier to find events/errors when trawling the logs 
- To enable us to be able to do analysis/correlation on certain errors. This becomes much much harder if the event string we are interested in is different every time.

With all that in mind I would change it to something like:
```
log.Event(ctx, "numeric search parameter provided with non numeric characters", log.Data{
    "param": "size",
    "value": sizeParam,
})
...
log.Event(ctx, "numeric search parameter provided with non numeric characters", log.Data{
    "param": "from",
    "value": paramGet(params, "from", "0"),
})
```
Now we have the same event for both invalid parameter scenarios and the specifics / variables of each case is available log.Data.

Sorry for the essay, I appreciate you weren't aware of this but I figured it would more useful to explain why we want this than just requesting a change with no context.

Reply:
That makes sense. Updated, as well as all instances of the incorrect spelling.